src/voters/services/voterOutreach.service.ts

```diff
@@ -123,6 +125,41 @@ export class VoterOutreachService {
       audienceRequest: audience['audience_request'],
     })
 
+    // If type is SMS, create a TextCampaign
+    if (type === 'sms') {
+      await this.textCampaignService.model.create({
```

Comment:
You shouldn't have to reference `.model` here.. 

```suggestion
      await this.textCampaignService.create({
```

Should work.. Not blocking tho